README.md

```diff
@@ -109,10 +109,6 @@ your project (for example, at `bin/venv-update`).
 Then, apply a change like this to your `tox.ini` file:
 
 ```patch
-  [tox]
-  envlist = py27,py34
-+ skipsdist = true
-
```

Comment:
That's false; if you look at tox -vvv, you can see that the sdist is installed with `--no-deps`.

```
$ tox -vvvve python
  removing .tox/log
using tox.ini: tox.ini
using tox-2.3.1 from venv/local/lib/python2.7/site-packages/tox/__init__.pyc
GLOB start: packaging
GLOB sdist-make: setup.py
  removing .tox/dist
  $ venv/bin/python2.7 setup.py sdist --formats=zip --dist-dir .tox/dist >.tox/log/tox-0.log
GLOB finish: packaging after 0.53 seconds
copying new sdistfile to '/nail/home/buck/.tox/distshare/pip-faster-1.0rc6.dev3.zip'
python start: getenv .tox/python
python reusing: .tox/python
python finish: getenv after 0.09 seconds
python start: installpkg .tox/dist/pip-faster-1.0rc6.dev3.zip
python inst-nodeps: .tox/dist/pip-faster-1.0rc6.dev3.zip
setting PATH=<snip>
  $ pip install -U --no-deps .tox/dist/pip-faster-1.0rc6.dev3.zip
```